echemdb/data/cv/entry.py

```diff
@@ -175,197 +190,148 @@ def names(_, context, role):
                     persons = context["entry"].persons[role]
                     style = context["style"]
 
-                    names = [style.format_name(person, style.abbreviate_names) for person in persons]
+                    names = [
+                        style.format_name(person, style.abbreviate_names)
+                        for person in persons
+                    ]
 
                     if len(names) == 1:
                         return names[0].format_data(context)
                     else:
                         from pybtex.style.template import words, tag
-                        return words(sep=' ')[names[0], tag('i')['et al.']].format_data(context)
+
+                        return words(sep=" ")[names[0], tag("i")["et al."]].format_data(
+                            context
+                        )
 
                 names = names(role)
 
                 from pybtex.style.template import sentence
+
                 return sentence[names] if as_sentence else names
 
             def format_title(self, e, which_field, as_sentence=True):
                 from pybtex.style.template import field, tag, sentence
-                title = tag('i')[field(which_field)]
+
+                title = tag("i")[field(which_field)]
                 return sentence[title] if as_sentence else title
 
-        return EchemdbStyle(abbreviate_names=True).format_entry("unused", self.bibliography).text.render_as(backend)
+        return (
+            EchemdbStyle(abbreviate_names=True)
+            .format_entry("unused", self.bibliography)
+            .text.render_as(backend)
+        )
 
-    def x(self):
-        r"""
-        Return the name of the variable on the x-axis, i.e., ``"E"``.
+    def field_unit(self, field_name):
+        """Returns the unit of a given field name of the `echemdb` resource.
 
         EXAMPLES::
 
             >>> entry = Entry.create_examples()[0]
-            >>> entry.x()
-            'E'
+            >>> entry.field_unit('E')
+            'V'
 
         """
-        from astropy import units as u
-
-        axis = getattr(self.data_description, 'axes', {})['E']
+        return self.package.get_resource('echemdb').schema.get_field(field_name)['unit']
 
-        if not u.Unit(axis.unit).is_equivalent('V'):
-            raise ValueError("The variable on the x-axis denoted as 'E', is not convertible to 'V'.")
-        
-        return 'E'
-
-    def y(self):
-        r"""
-        Return the name of the variable on the y-axis, i.e., ``"j"`` or ``"I"``.
-
-        EXAMPLES::
+    def rescale_original(self):
```

Comment:
I think it's easier to understand what this does if the call is just `rescale('original')`, i.e., if we make this part of the `rescale()` method. Is there a strong reason to separate this?